tools/format/format.cpp

```diff
@@ -203,7 +203,7 @@ int main( int argc, char *argv[] )
             std::ofstream fout( filename, std::ios::binary | std::ios::trunc );
             fout << out.str();
             fout.close();
-            std::cout << "Formatted " << filename << std::endl;
+            std::cout << filename << " needs to be linted" << std::endl;
```

Comment:
Regarding

> Not doing this, as it can indicate that the contributor hasn't read the JSON style guide.

Might I suggest explicitly mentioning this in the failure text? Some users may not even know the style guide exists.

```suggestion
            std::cout << filename << " needs to be linted"
	                       "\nPlease read doc/JSON_STYLE.md" << std::endl;
```

Reply:
Oh, I didn't see this, sorry.
I don't have any problems doing it
Though I would have done
```suggestion
            std::cout << filename << " needs to be linted" << std::endl;
            std::cout << "Please read doc/JSON_STYLE.md" << std::endl;
```
If you'd like to PR it, feel free.

Reply:
No worries. And fair enough, that is way cleaner.